flasc/utilities/floris_tools.py

```diff
@@ -357,8 +362,7 @@ def calc_floris_approx_table(
             ti_array=np.arange(0.03, 0.1801, 0.03)
             save_turbine_inflow_conditions_to_df=True
 
-        Yields:
-
+    Yields:
```

Comment:
What is this for? Seems like this should go in the main description, not after the `Returns:` list? Or maybe this is another standard field that I'm not familiar with?

Reply:
I'm actually not sure on this one, I vote to delete it